corpus/references/README.md

```diff
@@ -0,0 +1,65 @@
+# References
+
+This directory contains references used within the corpus. The sub-directory `bibfiles/` contains `.bib` files for each reference. The file `references.bib` is a compiled single reference file for the `.bib` files contained in the `bibfiles/` directory. `references.bib` is compiled using `scripts/references/compile-bib-references.py`.
```

Comment:
I agree that we shouldn't have duplicate data. A `references.bib` file would be sufficient for the corpus. *But* I don't think it's a good idea to add directly to that file (if you've ever had the pleasure of searching for error in 1000s of lines of bibtex, you know) -- having individual bib files is an easy way to manage additions and verify valid bibtex when compiling the `references.bib` file. Would you be happy to move the `bibfiles/` dir out of `corpus/'?

Reply:
Ok. Then maybe we keep the references as separate files and ad your combination script to the library? Then we can combine it if needed and only store separate bibtex files? I just want to avoid duplicated data being commited.